### PR TITLE
feat(nfts): add in markdown lib for collection descriptions

### PR DESCRIPTION
**Pull request by rsteubs-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/index.tsx [hunk 1]
```diff
@@ -232,12 +233,20 @@ const NftsCollection: React.FC<Props> = ({
           </LinksContainer>
           <Text
             lineHeight='1.4'
-            style={{ marginTop: '16px' }}
+            style={{ marginTop: '16px', maxHeight: '15em', overflow: 'hidden' }}
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/index.tsx:236]: can we expand this like opensea?

File: package.json [hunk 2]
```diff
@@ -221,5 +221,8 @@
   },
   "resolutions": {
     "qr.js": "https://github.com/defunctzombie/qr.js.git"
+  },
+  "dependencies": {
+    "marked": "^4.0.12"
```
Comment (plondon) [inline package.json:226]: This should be added to blockchain-wallet-v4-frontend dependency list

Comment (plondon) [reply package.json:226]: Please pin the dependency as well (no ^)

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/index.tsx (hunk 1 above)
Comment (rsteubs-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/index.tsx]: yep added toggle

File: packages/blockchain-wallet-v4-frontend/package.json [hunk 3]
```diff
@@ -124,6 +124,7 @@
     "lottie-web": "5.7.11",
     "lz-string": "1.4.4",
     "matchmedia-polyfill": "0.3.2",
+    "marked": " 4.0.12",
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/package.json:127]: fix extra space

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/index.tsx [hunk 4]
```diff
@@ -230,15 +236,50 @@ const NftsCollection: React.FC<Props> = ({
...
+              size='14px'
+              color='grey600'
+              style={{ cursor: 'pointer' }}
+              weight={500}
+              onClick={toggleDescription}
+            >
+              <div
+                id='preview'
+                // eslint-disable-next-line react/no-danger
+                dangerouslySetInnerHTML={{
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/index.tsx:251]: @kristovatlas thoughts on adding this markdown library for parsing markdown?

**Comment by plondon** [comment]
Designs removed collection description so gonna kill this

**Pull request closed without merging**